src/sass/vic-v2.scss

```diff
@@ -372,11 +372,10 @@ input[type=range] {
 }
 
 .drag-active-text {
-  position: absolute;
-  width: 240px;
-  height: 240px;
-  padding-top: 110px;
-  text-align: center;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
Noticed a minor layout issue caused by some bad assumptions on my part. Updated to use flex box.


<img width="467" alt="screen shot 2018-02-19 at 08 29 09" src="https://user-images.githubusercontent.com/4998130/36382763-b4daace8-154f-11e8-9994-0edcf4e74ab3.png">